ui/lib/core/addon/components/replication-summary-card.js

```diff
@@ -0,0 +1,52 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import layout from '../templates/components/replication-summary-card';
+
+/**
+ * @module ReplicationSummaryCard
+ * ReplicationSummaryCard components
+ *
+ * @example
+ * ```js
+ * <ReplicationSummaryCard
+    @title='States'
+    @replicationDetails=replicationDetails
+    />
+ * ```
+ * @param {string} [title=null] - The title to be displayed on the top left corner of the card.
+ * @param replicationDetails=null{DS.Model.replicationDetails} - An Ember data object off the Ember data model.  It is computed at the parent component and passed through to this component.
```

Comment:
The formatting of this one looks a little funky 🤔 is that right?

Reply:
nice catch! chelsea is right, the format should match the example in the README here:
https://github.com/hashicorp/vault/blob/dd81d580622050357f3e2ec90e51d30c77cb99f4/ui/README.md#L157

Reply:
while you're at it, i'd recommend updating this description to include a little more about what the contents of `replicationDetails` are. what information does it provide?